src/features/canvass/components/CanvassInstructionsPage.tsx

```diff
@@ -105,13 +107,17 @@ const Page: FC<{
           >
             <Button
               fullWidth
-              onClick={() => router.push(`/canvass/${assignment.id}/map`)}
+              onClick={() =>
+                areas.length > 1
```

Comment:
Since this condition is used twice, I think it would make sense to give it a name, e.g. `const userMustSelectArea = areas.length > 1`.

Reply:
Fixed in bee5625a01f5f6d2dc7675bea5181cdd24c1c2bc